src/services/registration.js

```diff
@@ -1,4 +1,4 @@
-const dbService = require('../services/regCheckDB');
+const dbService = require('./regCheckDB');
 const users = require('../dummy/userDB');
 
 const createUser = item => new Promise((resolve) => {
```

Comment:
resolve in else should be reject?
for naming: item could be newUser, since the function called createUser